modules/storages/spec/factories/storage_factory.rb

```diff
@@ -94,6 +85,13 @@
     end
   end
 
+  factory :nextcloud_storage,
```

Comment:
And I moved most traits to the storage parent as both storage types have automatically managed project folders
@akabiru